src/site/content/en/metrics/fcp/index.md

```diff
@@ -29,7 +29,7 @@ starts loading to when any part of the page's content is rendered on the screen.
 For this metric, "content" refers to text, images (including background images),
 `<svg>` elements, or non-white `<canvas>` elements.
 
-[![FCP timeline from google.com](fcp-filmstrip.png)](fcp-filmstrip.png)
+[{% Img src="image/admin/3UhlOxRc0j8Vc4DGd4dt.png", alt="FCP timeline from google.com](fcp-filmstrip.png)", width="800", height="311" %}
```

Comment:
I noticed this image has extra square brackets. It's actually a clever trick by Phil to wrap the image in an anchor so you can click on it and get the high res version (sort of like a lightbox).

I think all of the pages in /metrics/ use this technique (but I haven't seen it anywhere else).

In the short term I think we can put an `<a>` around the shortcode which links to a version of the image.
But I think this is a cool idea and maybe we should consider adding it to the shortcode at some point?